src/main/java/de/craften/plugins/rpgplus/scripting/api/entities/EntityModule.java

```diff
@@ -176,7 +176,10 @@ public LuaTable getNearbyEntities(LuaValue locationParam, LuaValue radius) {
 
     public RpgPlusEntity getEntity(Entity entity) {
         for (RpgPlusEntity rpgPlusEntity : entities) {
-            if (rpgPlusEntity.getEntity() == entity) {
+        	if (rpgPlusEntity == null || entity == null || rpgPlusEntity.getEntity() == null) {
```

Comment:
`entity` is never `null` unless this function is not used correctly, in which case an `IllegalArgumentException` should be thrown.